Fix #17351: Grant review translation and review question rights to admin user

aroch17 opened the pull request:
## Overview
<!--
READ ME FIRST:
Please answer *both* questions below and check off every point from the Essential Checklist!
If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
-->

1. This PR fixes or fixes part of #17351 
2. This PR does the following: Grants review translation and review question rights to the super admin user 'a' when ```python -m scripts.populate_sample_contributor_data``` is run.

## Essential Checklist

- [x] The **PR title** starts with "Fix #bugnum: " or "Fix part of #bugnum: ...",
followed by a short, clear summary of the changes.
- [x] I have followed the [instructions for making a code change](https://github.com/oppia/oppia/wiki/Make-a-pull-request).
- [x] The linter/Karma presubmit checks have passed on my local machine.
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [x] My PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
- [x] I have assigned the correct reviewers to this PR (or will leave a
comment with the phrase "@{{reviewer_username}} PTAL" if I don't have
permissions to assign reviewers directly).

## Proof that changes are correct

<!--
Add videos/screenshots of the user-facing interface to demonstrate that the changes
made in this PR work correctly. If this PR is for a developer-facing feature,
provide the videos/screenshots of developer-facing interface. Please also include
videos/screenshots of the developer tools browser console, so that we can be
sure that there are no console errors.

When you make updates to the PR, please update these videos/screenshots as well.
You can drop videos/screenshots from previous versions of the PR.

The above should be done for all PRs, including short ones (e.g. a single-line change).
However, if the changes in your PRs are autogenerated via a script and you cannot
provide proof for the changes then please leave a comment "No proof of changes
needed because {{Reason}}" and remove all the sections below.
-->

#### Proof of changes on desktop with slow/throttled network

<!--
Make sure to properly verify that everything works correctly, and that there are
no weird UI mistakes or other problems. Also, if there are any newly added fields,
try to fill them out and test that different inputs are correctly accepted/rejected.

Throttle the network (to 3G) using the browser Developer Tools (see references below).
There should be no performance or UI issues while the network is slow.

References:
 - Chrome: https://css-tricks.com/throttling-the-network/
 - Firefox: https://developer.mozilla.org/en-US/docs/Tools/Network_Monitor/Throttling
-->

## PR Pointers

- Never force push! If you do, your PR will be closed.
- To reply to reviewers, follow these instructions: https://github.com/oppia/oppia/wiki/Make-a-pull-request#step-5-address-review-comments-until-all-reviewers-approve
- Some e2e tests are flaky, and can fail for reasons unrelated to your PR. We are working on fixing this, but in the meantime, if you need to restart the tests, please check the ["If your build fails" wiki page](https://github.com/oppia/oppia/wiki/If-CI-checks-fail-on-your-PR).
- See the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia's-code-owners-and-checks-to-be-carried-out-by-developers) for what code owners will expect.

aroch17 committed:
Grant review translation rights to admin user

* This commit grants review translation rights of the language Akan to
the super admin user.

aroch17 committed:
Grant review question rights to the admin user

aroch17 committed:
Fix failing lint checks

reactor0533 commented:
@chris7716 PTAL
I haven't been able to include 'Proof that changes are correct' since I am still encountering the `ModuleNotFoundError`, but the code worked before the error started.

DubeySandeep commented on scripts/populate_sample_contributor_data.py:
```diff
@@ -101,6 +101,8 @@ def populate_data(self) -> None:
         self._sign_in(SUPER_ADMIN_EMAIL)
         self.csrf_token = self._get_csrf_token()
         self._assign_admin_roles(SUPER_ADMIN_ROLES, SUPER_ADMIN_USERNAME)
+        self._add_review_translation_rights(SUPER_ADMIN_USERNAME)
+        self._add_review_question_rights(SUPER_ADMIN_USERNAME)
```
Can we add a test to ensure users are assigned to the correct role?

DubeySandeep commented on scripts/populate_sample_contributor_data.py:
```diff
@@ -101,6 +101,8 @@ def populate_data(self) -> None:
         self._sign_in(SUPER_ADMIN_EMAIL)
         self.csrf_token = self._get_csrf_token()
         self._assign_admin_roles(SUPER_ADMIN_ROLES, SUPER_ADMIN_USERNAME)
+        self._add_review_translation_rights(SUPER_ADMIN_USERNAME)
```
Pass langauge_code as param! This will make it easier to understand what role is assigned to the user just by reading this section. (OR change the method name to `_add_review_akan_translation_rights`)

DubeySandeep commented on scripts/populate_sample_contributor_data.py:
```diff
@@ -207,6 +209,31 @@ def _assign_admin_roles(self, roles: List[str], username: str) -> None:
+        }
+
+        self._make_request(
+            'POST', '/contributionrightshandler/translation', params=params)
```
I think we should assign role on PUT

DubeySandeep replied:
Unexpectedly we are doing it via POST! (You can ignore this comment)

DubeySandeep reviewed (changes requested):
@aroch17 Thanks for the PR, I've left a few comments PTAL!

aroch17 committed:
Modify _add_review_translation_rights function

* This commit modifies the implementation of the
_add_review_translation_rights function by adding a parameter for
the language code.

aroch17 committed:
Fix failing lint checks

aroch17 committed:
Merge branch 'develop' into grant_review_translation_and_review_question_rights

aroch17 committed:
Merge branch 'develop' into grant_review_translation_and_review_question_rights
